gp-copy-cat/gpcc-copy-label.js

```diff
@@ -8,16 +8,26 @@
 * Use this snippet to copy the label instead of the value.
 *
 * Instructions:
-* 1. Install our free Custom Javascript for Gravity Forms plugin. 
+* 1. Install our free Custom Javascript for Gravity Forms plugin.
 *    Download the plugin here: https://gravitywiz.com/gravity-forms-custom-javascript/
 * 2. Copy and paste the snippet into the editor of the Custom Javascript for Gravity Forms plugin.
 */
 gform.addFilter( 'gppc_copied_value', function( value, $elem, data ) {
     $source = jQuery( '#input_' + data.sourceFormId + '_' + data.source );
     if( $source.is( 'select' ) ) {
         value = $source.find( 'option:selected' ).text();
-    }  else if( $source.is( '.gfield_radio' ) ) {
+    } else if( $source.is( '.gfield_radio' ) ) {
         value = $( '.gfield-choice-input:checked + label' ).text();
+	} else if ( $source.is( '.gfield_checkbox') ) {
+		// Since multiple checkboxes maybe selected, make sure individual checkbox values do not have whitespaces.
+		values = value.split( ' ' );
```

Comment:
This feels a bit flimsy. Can you use something like `$source.find( 'input:checked' )` to get the checked inputs? 

Then, you can use a jQuery `.each()` on it and get the sibling label and put that into the `labelArray`.